bootcamp/tutorials/integration/milvus_with_JINA.ipynb

```diff
@@ -0,0 +1,1205 @@
+{
```

Comment:
<p>clear the output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1328/discussion/#comment-1609395057'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
solved